docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -939,7 +939,7 @@ amount is received, the system should reject that loan application with some des
 ==== Write the Missing Implementation
 
 At some point in time, you need to send a request to the Fraud Detection service. Assume
-that you need to send the request containing the ID of the client and the amount the
+that you need to send the request containing the ID of the client, and the amount the
```

Comment:
Do not add this comma. Again, this conjunction ("and") is creating a two-item list rather than attaching an independent clause.